README.md

```diff
@@ -1,17 +1,40 @@
-# mpmp
+# *mpmp*: *M*ultimodal *P*an-cancer *M*utation *P*rediction
 
-**M**ultimodal **P**an-cancer **M**utation **P**rediction
+**Jake Crawford, Brock C. Christensen, Maria Chikina, Casey S. Greene**
 
-**Note:** This repository is currently a work in progress, so some aspects of the code/analysis may not be fully described or documented here.
+**University of Pennsylvania, University of Colorado Anschutz Medical Campus, Dartmouth College, University of Pittsburgh**
 
-Many lines of evidence suggest that driver mutations in cancer can affect normal cellular function through a wide variety of perturbations.
-These can include altered gene expression, altered DNA methylation patterns, and altered miRNA function, among others.
+Manuscript (in progress) is located here: https://greenelab.github.io/mpmp-manuscript/
 
-Thus, if our goal is to identify samples with a given driver mutation, it is plausible that different data modalities will contain different amounts of information.
-For example, IDH1 mutations have been [implicated in aberrant DNA methylation](https://doi.org/10.1038/s41598-019-53262-7), and TP53 mutation has been [shown to lead to distinct and detectable gene expression changes]( https://doi.org/10.1016/j.celrep.2018.03.076).
-Our goal is to develop a data-driven methodology for identifying and comparing the strength of relationships between various drivers and one or more data modalities.
+In studies of cellular function in cancer, researchers are increasingly able to choose from many -omics assays as functional readouts.
+Choosing the correct readout for a given study can be difficult, and it is not always clear which layer of cellular function is most suitable to capture the relevant signal.
```

Comment:
Relevant signal for what?

Reply:
Reworded this slightly (hopefully that will help clarify). I think we want to keep it fairly general here.